nibabel/pkg_info.py

```diff
@@ -85,20 +96,25 @@ def pkg_commit_hash(pkg_path=None):
     hash_str : str
        short form of hash
     """
-    versions = _version.get_versions()
-    hash_str = versions['full-revisionid'][:7]
-    if hasattr(_version, 'version_json'):
-        hash_from = 'installation'
-    elif not _version.get_keywords()['full'].startswith('$Format:'):
-        hash_from = 'archive substitution'
-    elif versions['version'] == '0+unknown':
-        hash_from, hash_str = '(none found)', '<not found>'
-    else:
-        hash_from = 'repository'
-    return hash_from, hash_str
-
-
-def get_pkg_info(pkg_path):
+    from subprocess import run
```

Comment:
Is there any benefit to importing this within the function rather than at the top of the module? AFAIK, as long as there are no dependency issues or critical performance considerations, the latter is generally encouraged.

Reply:
The idea was that we don't import subprocess outside this very rarely-used function, but we do import numpy which imports subprocess, so I guess it makes no difference.